src/main/resources/plugin.yml

```diff
@@ -14,11 +16,19 @@ commands:
     description: "Gives the player an additional creative plot."
     usage: "/givecreativeplot <player>"
     permission: mythicstore.giveplot
+  skincontrol:
+    description: "Main command for SkinControl."
+  runonjoin:
+    aliases:
+      - roj
```

Comment:
Give these commands a description, usage and permission like the other previously existing ones.